docs/source/usage.rst

```diff
@@ -93,43 +93,54 @@ characters) entered at the command prompt...
 Using a Packed Catalog
 ----------------------
 
-You can run databroker on top of a directory generated by ``databroker-pack``
-like so:
+Use ``databroker-unpack`` to make ``DIRECTORY`` automatically discoverable by
+databroker. You must specify a unique name to give the catalog.
 
-.. code:: python
+.. code:: bash
 
-   import intake
-   catalog = intake.open_catalog('DIRECTORY/catalog.yml')
+   databroker-unpack DIRECTORY NAME
 
-replacing ``DIRECTORY`` with the path to the directory generated by
-``databroker-pack``. This will contain a catalog named ``'packed_catalog'``,
-which you can open like so.
+For example
+
+.. code:: bash
+
+   databroker-unpack path/to/directory_from_pack my_data
+
+Then the newly "unpacked" catalog (e.g. ``my_data``) will show in
+
+.. code:: bash
+
+   databroker --list-catalogs
```

Comment:
Is it `databroker` or `databroker-pack`?

Reply:
For reference: https://github.com/bluesky/databroker-pack/blob/f970c72db82370f7216e14cf597434b548c3a6c6/databroker_pack/commandline/pack.py#L96

Reply:
@danielballan, thanks for fixing it via 48417b3.